src/main/java/org/mockito/CheckReturnValue.java

```diff
@@ -0,0 +1,23 @@
+package org.mockito;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+/**
+ * Findbugs handles any annotation with name "CheckReturnValue" in return value check.
+ *
+ * @see <a href="https://github.com/findbugsproject/findbugs/blob/264ae7baf890d2b347d91805c90057062b5dcb1e/findbugs/src/java/edu/umd/cs/findbugs/detect/BuildCheckReturnAnnotationDatabase.java#L120">Findbugs source code</a>
+ * @see <a href="http://errorprone.info/bugpattern/CheckReturnValue">ErrorProne check</a>
+ */
+@Target({
+    ElementType.CONSTRUCTOR,
+    ElementType.METHOD,
+    ElementType.PACKAGE,
+    ElementType.TYPE
+})
+@Retention(RetentionPolicy.CLASS)
```

Comment:
Shouldn't it be RetentionPolicy.RUNTIME? Annotations with RetentionPolicy.CLASS are not preserved for the vm runtime execution, which is what I assume Findbugs needs.

Reply:
I don't think so, it seems to work in AssertJ: https://github.com/karlicoss/assertj-core/blob/master/src/main/java/org/assertj/core/util/CheckReturnValue.java

Reply:
ok :)